test-resources/service-specs/v4/API_TEST_SRV/API_TEST_SRV.edmx

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
General question: When I run `npm run generate:test-services`. I thought it will generate the services for v2 and v4 given in these files and generate them also coping the files to the `packages/core/test/test-util/test-services/v4/test-service/` and `packages/core/test/test-util/test-services/v2/test-service/`. I think this is what happening but I wonder why there is no change in the entities? Is the v4 edmx files designed in a way to keep the existing code?

Reply:
Yes, but not completely. You won't have changes if I already re-generated, though.